docs/csharp/misc/cs0460.md

```diff
@@ -10,13 +10,17 @@ ms.assetid: 98d39ded-d3f9-4520-b912-892e574c056b
 ---
 # Compiler Error CS0460
 
-Constraints for override and explicit interface implementation methods are inherited from the base method, so they cannot be specified directly  
+Constraints for override and explicit interface implementation methods are inherited from the base method, so they cannot be specified directly, except in specific cases  
```

Comment:
This change should be reverted. It's the verbatim error message, and it's repeated for SEO usage:

```suggestion
Constraints for override and explicit interface implementation methods are inherited from the base method, so they cannot be specified directly
```